packages/core/src/external/athenahealth/index.ts

```diff
@@ -199,7 +199,21 @@ class AthenaHealthApi {
           })
           .catch(processAsyncError("Error saving to s3 @ AthenaHealth - getPatient"));
       }
-      return patientResourceSchema.parse(response.data);
+      const patient = patientResourceSchema.safeParse(response.data);
+      if (!patient.success) {
+        capture.message(`Patient could not be parsed @ AthenaHealth`, {
```

Comment:
nit: the "@ Athena" is confusing since we're parsing it on our end. Maybe "Patient from Athena could not be parsed" or something similar?

Also, I suggest we log in addition to sending to Sentry

Reply:
Done.